src/plugins/data/public/ui/query_editor/editors/default_editor/index.tsx

```diff
@@ -66,22 +66,23 @@ export const DefaultInput: React.FC<DefaultInputProps> = ({
           ],
         }}
       />
-      {footerItems && (
-        <EuiFlexGroup direction="row" className="defaultEditor__footer">
-          <EuiFlexGroup direction="row">
+      <div className="defaultEditor__footer">
+        {footerItems && (
+          <EuiFlexGroup direction="row" alignItems="center">
             {footerItems.start?.map((item) => (
-              <EuiFlexItem grow={false} color="red" className="defaultEditor__footerItem">
+              <EuiFlexItem grow={false} className="defaultEditor__footerItem">
                 {item}
               </EuiFlexItem>
             ))}
-          </EuiFlexGroup>
-          <EuiFlexGroup direction="row" className="defaultEditor__footerItem__end">
+            <EuiFlexItem grow />
             {footerItems.end?.map((item) => (
-              <EuiFlexItem grow={false}>{item}</EuiFlexItem>
+              <EuiFlexItem grow={false} className="defaultEditor__footerItem">
```

Comment:
do we still need this className `defaultEditor__footerItem`? i see it got deleted

Reply:
No, but keeping it in case we do